octokit-project/src/main/java/org/example/Main.java

```diff
@@ -0,0 +1,22 @@
+package org.example;
+
+public class Main {
+
+    public static void main(String[] args) {
```

Comment:
[Severity: Low] The variable names are not descriptive enough.
Consider using more meaningful names for variables (e.g., 'numberA' instead of 'a') to enhance code clarity.
Rationale: Descriptive variable names help other developers understand the code's purpose without needing additional comments.
Reference: https://refactoring.guru/replace-parameter-with-explaining-variable